selfdrive/car/tests/test_car_interfaces.py

```diff
@@ -123,7 +123,10 @@ def test_car_interfaces(self, car_name, data):
     if not car_params.radarUnavailable and radar_interface.rcp is not None:
       cans = [messaging.new_message('can', 1).to_bytes() for _ in range(5)]
       rr = radar_interface.update(cans)
-      self.assertTrue(rr is None or len(rr.errors) > 0)
+      self.assertTrue(len(rr.errors) > 0)
+
+      rr = radar_interface.update([])
+      self.assertTrue(len(rr.errors) > 0)
```

Comment:
I would assert it's false, then assert it's true after a single run

Reply:
can you elaborate? `rr` is not defined before running update

Reply:
ah I see, perhaps then just switch these two conditions so it first runs on an empty list